examples/chartmuseum/componentplan_auth.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: ComponentPlan
+metadata:
+  name: chartmuseum
+  namespace: kubebb-system
+spec:
+  approved: true
+  name: chartmuseum
+  version: 3.10.1
+  override:
+    set:
+      - env.secret.BASIC_AUTH_USER=admin 
+      - env.secret.BASIC_AUTH_PASS=password
```

Comment:
According to my test, deploy without setting `env.open.DISABLE_API=false`. Then the `api of charmuseum` is not available to upload the chart package via curl command.